tests/test_integration.py

```diff
@@ -46,6 +63,28 @@
     AutoEncodingTopicModel(5, combined=True),
 ]
 
+dynamic_models = [
+    GMM(5, encoder=trf),
+    ClusteringTopicModel(
```

Comment:
Isn't this a bit too many? It will take a looooong time to run tests.

Reply:
i've removed one -- but it might still be good to run both centroid and one of the c-tf-idf options since the logic is so different?

Reply:
Yeah let's do that but then maybe remove some of them from the static model test?